### DISCUSSION TITLE
Class "UserSearch" not found

### DISCUSSION THREAD

**Discussion by williamsdb** [noweh/twitter-api-v2-php#32] [Q&A]
I am trying to get the number of followers associated with a user. For this, I am calling findByIdOrUsername but the UserSearch::MODES['USERNAME'] part throws an error saying that UserSearch doesn't exist. What am I doing wrong please?

EDIT: If I remove that part of the code which seems possible as the code seems to work it out for itself then I get this error returned:

`Client error: GET https://api.twitter.com/2/users/by/username/williamsdbf1?user.fields=description resulted in a 401 Unauthorized response: { "title": "Unauthorized", "type": "about:blank", "status": 401, "detail": "Unauthorized" } `

<img width="1124" alt="Monosnap Whoops! There was an error  2023-05-24 16-48-03" src="https://github.com/noweh/twitter-api-v2-php/assets/8099483/a711bf37-736b-4be6-9f54-ef4e1db9a2d8">

**Comment by noweh** [comment]
In the current version (>=3.0.0), UserSearch became \Noweh\TwitterApi\UserLookup.
I will update the documentation accordingly.
Regards
